keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -2,6 +2,9 @@ package com.netflix.spinnaker.keel.api.artifacts
 
 typealias ArtifactType = String
 
+const val DEBIAN: ArtifactType = "deb"
+const val DOCKER: ArtifactType = "docker"
```

Comment:
Moved these constants here from in `keel-core` as `DEBIAN` is used as a default in `ServerGroupSpec`

Reply:
Right, but isn't `ClusterSpec` itself a resource-type-specific class that should not be in `keel-api`?